### DISCUSSION TITLE
Computed fields future capabilities

### DISCUSSION THREAD

**Discussion by whygee-dev** [prisma/prisma#18404] [Q&A]
The addition of computed fields was very much needed, however its usage in the current API are rather limited. We can't order by or filter by computed fields so my question is:

Will we eventually be able to orderBy or filter by a computed field ? We have an use case with a computed field involving some complex calculations that needs to be sorted and filtered

Our current workaround is to use an SQL view, but we aren't too keen on using raw SQL for such things

**Comment by ludralph** [comment]
Hi @whygee-dev 👋 

At the moment, filtering by computed fields is not supported. However, you can add your feedback to this [issue](https://github.com/prisma/prisma/issues/16500) so our team can take a look.

**Reply by ludralph** [reply]
Hey @whygee-dev ,

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by whygee-dev** [reply]
Hello @ludralph,

Well I did add a comment to the issue you've linked with no response yet. I guess I have to take a "no" for now and mark this question as answered.

Thank you
